src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -261,11 +566,37 @@ EscrowCreate::doApply()
         (*slep)[sfDestinationNode] = *page;
     }
 
-    // Deduct owner's balance, increment owner count
-    (*sle)[sfBalance] = (*sle)[sfBalance] - ctx_.tx[sfAmount];
+    // If issuer is not source or destination, add escrow to issuers owner
+    // directory.
```

Comment:
Assuming my understanding is correct, would be useful to add a comment for future readers:
```
// IOU escrow objects are added to the issuer's owner directory to help track the total locked balance.
// For MPT, this isn't necessary because the locked balance is already stored directly in the MPTokenIssuance object.
```